includes/definitions/discovery/procurve.yaml

```diff
@@ -90,3 +90,47 @@ modules:
                         - { descr: psFailed, graph: 1, value: 4, generic: 2 }
                         - { descr: psPermFailure, graph: 1, value: 5, generic: 2 }
                         - { descr: psMax, graph: 1, value: 6, generic: 3 }
+                -
+                    oid:  hpStackConfig
+                    value: hpStackOperStatus
+                    num_oid: '.1.3.6.1.4.1.11.2.14.11.5.1.69.1.1.2.0'
+                    descr: 'Stack State'
+                    state_name: hpStackOperStatus
+                    states:
+                        - { descr: disabled, graph: 1, value: 1, generic: 0 }
```

Comment:
First of all nice commit. I have the feeling that the `disabled/unAvailible` state should better be generic 3 (unknown). Generic 0 (OK) doesn't feel right to me.